src/scicookie/ui.py

```diff
@@ -96,11 +104,17 @@ def make_questions(questions: dict[str, Any]) -> dict[str, str]:
     )
     print("." * columns)
 
+    # Create a Jinja2 environment and add the custom filter
+    env = Environment(autoescape=True)
```

Comment:
please use autoescape=false.

if bandit complains, just skip the issue there, for example: https://github.com/osl-incubator/makim/blob/main/pyproject.toml#L105